client/mysql.cc

```diff
@@ -1632,6 +1637,10 @@ static struct my_option my_long_options[] =
    0, 0, 0, GET_NO_ARG, NO_ARG, 0, 0, 0, 0, 0, 0},
   {"binary-as-hex", 0, "Print binary data as hex", &opt_binhex, &opt_binhex,
    0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
+  {"bind-address", OPS_BIND_ADDRESS,
+   "name IP address to bind to.",
```

Comment:
`OPS_BIND_ADDRESS` can just be `0` (unless I've missed something significant on how this works).
Omit `"name '", this is filled out in help and `--bind-address=name name IP address to bind to.` looks quite odd.

Reply:
thanks for the notes. fixed